moped-database/metadata/tables.yaml

```diff
@@ -2614,6 +2614,7 @@
   event_triggers:
   - name: activity_log_moped_project
     definition:
+      enable_manual: false
```

Comment:
I'm seeing a lot of these `enable_manual` updates to several tables in this file and along with other trivial changes. I assume these metadata changes are mostly automated but I don't think we should commit changes unless they are related to the feature. Do you know what the `enable_manual` flag does? It seems like the only changes that should've been committed are the ones related to `moped_user_followed_projects` around line 3344. The rest just seems like unnecessary noise.

Reply:
This happened to me too once when i was changing things in the hasura console. Also often times all of the permissions are shuffled around if i change them via the console. It annoys the heck out of me, and I've manually changed the table so only the pertinent changes are applied.

Reply:
thanks for your perspective on this @mateoclarke and @chiaberry! i've been combing through this and agree the only relevant changes are related to permissions for the new `moped_user_followed_projects` table and everything else is just shuffling around existing information. i'm going to revert to the original file and manually add in the permissions updates and ping y'all when that's done!

Reply:
Going to mark this as resolved per our conversation during dev sync where we decided to go ahead and merge this file in as-is but be more selective about which changes to this file to commit in the future.